static/styles/rendered_markdown.css

```diff
@@ -138,7 +138,7 @@
     }
 
     .alert-word {
-        background-color: hsla(102, 85%, 57%, 0.5);
+        background-color: #691d1c;
```

Comment:
We use HSLA for all of our colors; the linter is probably complaining about this.

Also, probably we want different colors for the day and night themes (The latter in `night-mode.css`).

Reply:
> We use HSLA for all of our colors; the linter is probably complaining about this.

Thanks, Tim! I will change it now!

> Also, probably we want different colors for the day and night themes (The latter in night-mode.css).

Is it okay if I use a lighter shade of red for the day theme?